helpers/constants.js

```diff
@@ -91,5 +91,18 @@ module.exports = {
 	signatureLength: 196,
 	// WARNING: When changing totalAmount you also need to change getBlockRewards(int) SQL function!
 	totalAmount: 10000000000000000,
-	unconfirmedTransactionTimeOut: 10800 // 1080 blocks
+	unconfirmedTransactionTimeOut: 10800, // 1080 blocks
+
+	setConst: function (key, value) {
```

Comment:
getter is throwing errors when some property is unset, so the app will crash instead of working with undefined which will then could be a nightmare to track on